controllers/controllers.go

```diff
@@ -32,6 +32,10 @@ var log = logf.Log.WithName("controller")
 // DefaultCLITimeout is the default timeout for CLI commands.
 var DefaultCLITimeout = 10
 
+const (
+	clusterFile = "cluster-file"
```

Comment:
This feels like a name that might conflict with other variable names. Should we make it something more like `clusterFileKey`, kind of like what we have down with the constants for labels on pods?